src/RepairManager/main.py

```diff
@@ -0,0 +1,58 @@
+import time
+import sys
+import yaml
+import logging
+import logging.config
+import importlib
+
+import Rules
+
+
+with open('logging.yaml', 'r') as log_file:
+    log_config = yaml.safe_load(log_file)
+
+logging.config.dictConfig(log_config)
+logger = logging.getLogger('basic')
```

Comment:
Please use `logger = logging.getLogger(__name__)`. Also remove following setLevel, level is already set in logging.yaml config.